hrm-domain/packages/hrm-types/Contact.ts

```diff
@@ -67,7 +67,7 @@ export type NewContactRecord = {
 };
 
 export type ExistingContactRecord = {
```

Comment:
I think this type does not really means "record" as it's not the database shape, right? If so I'd love to rename this type (I'm happy to do so in a subsequent PR as that will cascade a ton of changes).